dev/com.ibm.ws.st.core/about.properties

```diff
@@ -20,5 +20,5 @@ Build id: {1}\n\
 \n\
 An eclipse IDE, for building and deploying Java EE, OSGi and Web 2.0 applications to WebSphere Application Server.\n\
 \n\
-Copyright \u00a9 IBM Corporation and others 2007, 2014. All rights reserved.\n\
+Copyright \u00a9 IBM Corporation and others 2007, 2018. All rights reserved.\n\
```

Comment:
Why is the first date here 2007 but the first date above is 2011?  If this is open liberty should the first date be 2017?